services/ui-src/src/components/forms/Form.tsx

```diff
@@ -48,7 +48,35 @@ export const Form = ({
   const { userIsAdmin, userIsReadOnly } = useStore().user ?? {};
 
   const { report } = useStore();
-  const targetPopulationChoiceList = report?.fieldData?.targetPopulations;
+
+  const updateRenderFields = (fields: (FormField | FormLayoutElement)[]) => {
+    const updatedTargetPopulationChoices = report?.fieldData?.targetPopulations;
+
+    const formatChoiceList = updatedTargetPopulationChoices?.map(
+      (field: EntityShape, key: string) => {
+        return {
+          checked: false,
+          id: field.id,
+          label:
+            parseInt(key) < 4
```

Comment:
what does the number 4 represent here? Might be beneficial to assign it to a variable to name it, as opposed to introducing a magic number here.

Reply:
if it's to check if it's one of those 4 default options...i wonder if we can check to see if it's actually one of the 4 default target pops, as opposed to one of the first 4 items in the array.  Maybe create (or utilize, if it already exists) an array with those 4 options and check if the string exists in that array

Reply:
Good point. It's for the 4 default target populations that are already generated. I can make that a variable and use the initial array.